docs/changehistory/NextVersion.md

```diff
@@ -13,6 +13,8 @@ ignore: true
 
 * The type of [Texture.data]($backend) has been corrected from `string` to `Uint8Array` to match the type in the BIS schema. If you get compilation errors, simply remove calls to `Buffer.from(texture.data, "base64")` for read, and `texture.data.toString("base64")` if you create texture objects.
 
+* The class [ToolUiManager]($ui-framework) has been renamed to [ToolSettingsManager]. 
```

Comment:
@bsteinbk is this a recent change or was it in 2.9?  I don't see the changes in this PR.  If it was 2.9, mind moving it to the changelog for that file?

Reply:
This probably came in via a merge with master branch. Where would you like it moved, @calebmshafer ?

Reply:
Can you add it to `2.9.0.md`?  We can then cherry-pick it back to the `release/2.9.0` branch.

Reply:
I was looking at the wrong part of the diff. I moved the ToolUiManager line to the 2.9.0.md